docs/packages/Parser.mdx

```diff
@@ -355,3 +355,34 @@ module.exports = {
   },
 };
 ```
+
+### `withoutProjectParserOptions(parserOptions)`
+
+Removes options that prompt the parser to parse the project with type
+information. In other words, you can use this if you are invoking the parser
+directly, to ensure that one file will be parsed in isolation, which is much,
+much faster.
+
+This is useful in cases where you invoke the parser directly, such as in an
+ESLint plugin context.
+
```

Comment:
Don't wrap lines by width. Either no breaks or wrap at sentence stops.

Reply:
Thanks for catching this! Changed to sentence stops in 15d47de3c351ef4e1d1fb836dbc69002330733bd